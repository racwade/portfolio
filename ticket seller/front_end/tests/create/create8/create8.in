login
test_admin
create
simpleName
buying and selling selected
logout