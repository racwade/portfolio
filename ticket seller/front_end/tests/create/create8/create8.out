welcome to the ticket seller system
enter username:
logged in as standard_admin
enter account username:
enter account type (admin/buying/selling):
error: Too many account types selected.
bye